SignalShareExtension/ShareViewController.swift

```diff
@@ -664,10 +673,75 @@ public class ShareViewController: UIViewController, ShareViewDelegate, SAEFailed
         // FIXME: For now, we prefer the URL provider and discard the text provider, since it's more useful to share the URL than the caption
         // but we *should* include both. This will be a bigger change though since our share extension is currently heavily predicated
         // on one itemProvider per share.
```

Comment:
If we wanted to handle what this comment is talking about, we could do that now.